### PR TITLE
[MU3] Sync vtest script for run on CI

### PR DESCRIPTION
**Pull request by igorkorsukov**
Sync vtest script for run on CI

### CODE REVIEW

File: .github/workflows/ci_vtests.yml [hunk 1]
```diff
@@ -17,51 +17,33 @@ jobs:
...
-        echo "PATH=$PATH" >> $GITHUB_ENV
-        echo "QT_PLUGIN_PATH=$QT_PLUGIN_PATH" >> $GITHUB_ENV
-        echo "QML2_IMPORT_PATH=$QML2_IMPORT_PATH" >> $GITHUB_ENV
-        git checkout -- build/travis/job1_Tests/environment.sh
+        sudo bash ./build/ci/linux/setup.sh
     - name: Build
       if: contains( env.found, '1')
       run: |
-        make installdebug CPUS=2 PREFIX="$HOME/software" COVERAGE=ON
+        . ./../musescore_environment.sh
```
Comment (Jojo-Schmitz) [inline .github/workflows/ci_vtests.yml:37]: the `.` command is pretty easy to get overlooked, maybe better use the `source` command?

Comment (igorkorsukov) [reply .github/workflows/ci_vtests.yml]: I agree. Changed.

Comment (Jojo-Schmitz) [reply .github/workflows/ci_vtests.yml]: This `. ./../` looks like ancient Cuneiform ;-)

**Comment by Jojo-Schmitz** [comment]
Quite funny vtest failure: no differences found but reported anyhow.

**Comment by igorkorsukov** [comment]
> Quite funny vtest failure: no differences found but reported anyhow.

some error png generation didn't find what's wrong yet

**Comment by AntonioBL** [comment]
Ciao. I think you could try to export QT_QPA_PLATFORM=offscreen or QT_QPA_PLATFORM=minimal:enable_fonts before the xvfb-run function.

**Comment by igorkorsukov** [comment]
> Ciao. I think you could try to export QT_QPA_PLATFORM=offscreen or QT_QPA_PLATFORM=minimal:enable_fonts before the xvfb-run function.

Thanks, I thought about it too.
But I want to understand why it used to work before and why it works in the master without these settings.

**Pull request merged**